flow/project.py

```diff
@@ -4498,6 +4500,15 @@ def _main_submit(self, args):
         aggregates = self._select_jobs_from_args(args)
 
         names = args.operation_name if args.operation_name else None
+
+        if kwargs.get("memory", None):
+            print(
+                "WARNING: "
+                "The memory argument is deprecated as of 0.13 "
+                "and will be removed in 0.14. Use directives to handle the "
+                "reservation of memory per node.",
+                file=sys.stderr,
+            )
```

Comment:
Use warnings.warn with a DeprecationWarning.

Reply:
This whole block is removed hence this change is no longer required